From c6c022c42e6b9115cbc36dce3f9100b90c2d2b06 Mon Sep 17 00:00:00 2001
From: Zoltan HERPAI <wigyori@uid0.hu>
Date: Tue, 20 May 2014 22:28:49 +0200
Subject: [PATCH] ARM: sun7i: dt: Add board support for LinkSprite pcDuino V3

The LinkSprite pcDuino V3 is an A20-based revision of the
earlier pcDuinos. This series will add support for the board,
along with some of its devices where the driver is accepted or
soon-to-be-accepted into mainline.

Changes since v2:
 - update MMC entry to comply with upstream
 - unify the 4 patches into one

Changes since v1:
 - fix cosmetic issues
 - fix i2c entry
 - remove unnecessary input bindings include
 - add MMC support

Signed-off-by: Zoltan HERPAI <wigyori@uid0.hu>
---
 arch/arm/boot/dts/Makefile               |   3 +-
 arch/arm/boot/dts/sun7i-a20-pcduino3.dts | 119 +++++++++++++++++++++++++++++++
 2 files changed, 121 insertions(+), 1 deletion(-)
 create mode 100644 arch/arm/boot/dts/sun7i-a20-pcduino3.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 6967393..b1b59ea 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -353,7 +353,8 @@ dtb-$(CONFIG_MACH_SUN7I) += \
	sun6i-a31-colombus.dtb \
 	sun7i-a20-cubieboard2.dtb \
 	sun7i-a20-cubietruck.dtb \
-	sun7i-a20-olinuxino-micro.dtb
+	sun7i-a20-olinuxino-micro.dtb \
+	sun7i-a20-pcduino3.dtb
 dtb-$(CONFIG_ARCH_TEGRA) += tegra20-harmony.dtb \
 	tegra20-iris-512.dtb \
 	tegra20-medcom-wide.dtb \
diff --git a/arch/arm/boot/dts/sun7i-a20-pcduino3.dts b/arch/arm/boot/dts/sun7i-a20-pcduino3.dts
new file mode 100644
index 0000000..fc6a542
-- 
2.0.3

